apps/emqx_ft/src/emqx_ft_app.erl

```diff
@@ -22,6 +22,7 @@
 
 start(_StartType, _StartArgs) ->
     {ok, Sup} = emqx_ft_sup:start_link(),
+    ok = emqx_ft_async_reply:create_tables(),
```

Comment:
Since this creates ETS tables, perhaps it would be safer to let the supervisor create and own the ETS tables?

That way, the ETS tables will be refreshed when the applications are restarted when joining a cluster.  I think the application master won't restart itself.

Reply:
✅